setup.py

```diff
@@ -0,0 +1,32 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="asaplib",
+    version="0.0.1",
+    author="Bingqing Cheng",
+    author_email="tonicbq@gmail.com",
+    description="Automatic Selection And Prediction tools for materials and molecules",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/BingqingCheng/ASAP",
+    packages=setuptools.find_packages(),
+    scripts=['scripts/krr.py',
```

Comment:
I'm not sure what the impact of moving these scripts from the `asap` folder to a designated `scripts` folder will be. Perhaps this is something for @BingqingCheng to comment on?

Reply:
I principle I'm not against moving the scripts to the scripts/ folder, as I see the logic in doing this.

This will make some example not running, as the path will be changed. I think one solution to this is 

1) not using the relative path in the examples, but just the scripts.
2) add a line in the README.md to explain how to install this, without the docker, i.e.
export PATH=~/repositories/ASAP/asap/:$PATH

Reply:
Should indeed update the readme about installation and the examples as well. 

However the fact that we install the package makes the scripts available in the path by default, so `python setup.py install` or `pip install .` just adds them to the python bin, so there is no need for exporting that. Earlier the `import asaplib` in the scripts imported something from the same dir, but now it is a site package that can be imported from anywhere.